pom.xml

```diff
@@ -551,6 +551,18 @@
             </properties>
         </profile>
         <profile>
+            <id>cloudera</id>
+            <properties>
+                <spark.version>2.0.0.cloudera2</spark.version>
```

Comment:
Hi

Which scala version is used in 2.0.0.cloudera2? The default scala version is scala 2.10. As spark 2.x pumped to scala 2.11, we change the scala version in spark_2.x profiles, e.g.
```xml
                 <scala.major.version>2.11</scala.major.version>
                 <scala.version>2.11.8</scala.version>
                 <scala.macros.version>2.1.0</scala.macros.version>
```
Do we need to do this for 2.0.0.cloudera2?